lib/ProductOpener/Web.pm

```diff
@@ -377,4 +378,33 @@ sub get_countries_options_list ($target_lc, $exclude_world = 1) {
 	return \@countries_list;
 }
 
+=head2 add_params( $url, $params )
+
+Adds parameters to a URL while properly handling the existing query string.
+
+=head3 Arguments
+
+=head4 $url - The URL to which parameters need to be added.
+
+=head4 $params - The parameters to be added in the format "key=value".
+
+=head3 Return value
+
+The modified URL with added parameters.
+
+=cut
+
+sub add_params ($url, $params) {
```

Comment:
@himanshisrestha we can even consider having


`sub add_params_to_link ($url, @params) {`

@params is a list of parameters you want to add.

This enables calling the function using:

`add_params_to_link($my_url, "test=foo", "othe=bar");` with as many parameters as we want.